README.md

```diff
@@ -20,6 +20,17 @@ Skapa en `env.json`-fil med följande för att få stripe-integrationen att fung
 }
 ```
 
+För att installera alla appens beroenden körs kommandot:
+```bash
+pip install -r requirements/dev.txt
+```
+För datalagring används en Postgres server som förväntas finnas på port `5432`.
```

Comment:
It also needs a user and password, no? Either way that should probably be explicit as to not confuse old devs like me 😉

Reply:
How do you set up the username password? I have forgotten, mid writing this part? :)

Reply:
I mean, does the app expect a certain user/pass? If so, then [`createuser`](https://www.postgresql.org/docs/current/static/app-createuser.html) is your friend.

Reply:
I do not know, I think we just used the setup created for the ticket system. Do you remember in the code we are specifying username/password?